### DISCUSSION TITLE
Output format in CoLab of data

### DISCUSSION THREAD

**Discussion by OneManShow0815** [mrdbourke/pytorch-deep-learning#973] [Q&A] [answered]
Some data gives just the data, some data gives data and dtype without specifically requesting it.
When does CoLab gives out the datatype with it?
![CoLab1 15 06 2024](https://github.com/mrdbourke/pytorch-deep-learning/assets/172930817/7c6ed101-5730-4a49-bd44-d5ff073d8251)

**Comment by miladnorouziii** [comment, marked as answer]
Hi, the data type (dtype) is typically displayed when you create a NumPy array or a Pandas DataFrame. You can also access the data type with *arr.dtype* command.

**Reply by OneManShow0815** [reply]
Thanks, great.
